urwid/event_loop/abstract_loop.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python
```

Comment:
this line only makes sense when we want a python file to be executable, right?

Reply:
Yep. This file was copied as-is, and then splitted.